metrics/web/test/common/presentation/navigation/metrics_page/metrics_page_factory_test.dart

```diff
@@ -2,167 +2,239 @@
 // that can be found in the LICENSE file.
 
 import 'package:metrics/auth/presentation/pages/login_page.dart';
-import 'package:metrics/common/presentation/navigation/constants/default_routes.dart';
 import 'package:metrics/common/presentation/navigation/metrics_page/metrics_page_factory.dart';
+import 'package:metrics/common/presentation/navigation/route_configuration/route_name.dart';
 import 'package:metrics/common/presentation/pages/loading_page.dart';
+import 'package:metrics/dashboard/presentation/models/dashboard_page_parameters_model.dart';
 import 'package:metrics/dashboard/presentation/pages/dashboard_page.dart';
 import 'package:metrics/debug_menu/presentation/pages/debug_menu_page.dart';
 import 'package:metrics/project_groups/presentation/pages/project_group_page.dart';
 import 'package:mockito/mockito.dart';
 import 'package:test/test.dart';
 
-import '../../../../test_utils/route_configuration_mock.dart';
 import '../../../../test_utils/route_name_mock.dart';
 
 void main() {
   group("MetricsPageFactory", () {
-    const path = 'path';
+    const unknownRouteName = 'unknown';
+    const pageParameters = DashboardPageParametersModel();
+    const dashboardRouteName = RouteName.dashboard;
+    const loadingRouteName = RouteName.loading;
+    const loginRouteName = RouteName.login;
+    const projectGroupsRouteName = RouteName.projectGroups;
+    const debugMenuRouteName = RouteName.debugMenu;
 
     final metricsPageFactory = MetricsPageFactory();
-    final routeConfiguration = RouteConfigurationMock();
     final routeName = RouteNameMock();
 
     tearDown(() {
-      reset(routeConfiguration);
       reset(routeName);
     });
 
     test(
-      ".create() returns the dashboard metrics page if the given route configuration is null",
+      ".create() returns the dashboard metrics page if the given route name is null",
```

Comment:
🏷️ 
```suggestion
      ".create() returns the metrics page with the dashboard page child if the given route name is null",
```
What do you think? Please, apply this suggestion to all similar situations.